### DISCUSSION TITLE
Connection error in Hiddify: "An established connection was aborted by the software in your host machine"

### DISCUSSION THREAD

**Discussion by this-xkit** [hiddify/hiddify-app#1459] [Q&A]
Hi! I’m encountering an error when using Hiddify. During connection, the following errors appear (see attached screenshot):

Error 1: An established connection was aborted by the software in your host machine
Error 2: An existing connection was forcibly closed by the remote host

Could you please advise on what might be causing these repeated connection terminations? The VPN is functioning, but these errors keep appearing every couple of minutes. Could it be related to network settings, firewall issues, or specific Hiddify configuration requirements? I would appreciate any guidance!

![Снимок экрана 2024-10-31 225546](https://github.com/user-attachments/assets/c5142508-bee7-477b-af7a-37178b983ff6)

**Comment by S-Vladislav** [comment]
Have you found a solution? I tried disabling microsoft defender, firewall, it didn't help. And yes my VPN is working as well (system proxy mode), it's just annoying.

**Reply by this-xkit** [reply]
The problem was solved after reinstalling Hiddify Manager and reinstalling Windows on my PC.
